packages/ra-core/src/form/useInput.ts

```diff
@@ -97,7 +97,7 @@ const useInput = ({
             id: id || source,
             input: options.input,
             meta: options.meta,
-            isRequired: isRequired(validate),
+            isRequired: options.isRequired || isRequired(validate),
```

Comment:
Would you mind moving the `isRequired` out of the options and directly in the `InputProps` as an optional boolean?

Reply:
I made the change, is it o like that?